cloudfoundry-client/src/main/lombok/org/cloudfoundry/client/v2/organizationquotadefinitions/UpdateOrganizationQuotaDefinitionRequest.java

```diff
@@ -0,0 +1,180 @@
+     * @return applicationInstanceLimit
+     */
+    @Getter(onMethod = @__(@JsonProperty("app_instance_limit")))
+    private final Integer applicationInstanceLimit;
+
+    /**
+     * The application task limit
+     *
+     * @param appTaskLimit the application task limit
+     * @return then application task limit
+     */
+    @Getter(onMethod = @__(@JsonProperty("app_task_limit")))
+    private final Integer applicationTaskLimit;
+
+    /**
+     * The instance memory limit
+     *
+     * @param instanceMemoryLimit the instance memory limit
+     * @return instanceMemoryLimit
+     */
+    @Getter(onMethod = @__(@JsonProperty("instance_memory_limit")))
+    private final Integer instanceMemoryLimit;
+
+    /**
+     * The memory limit
+     *
+     * @param memoryLimit the memory limit
+     * @return memory limit
+     */
+    @Getter(onMethod = @__(@JsonProperty("memory_limit")))
+    private final Integer memoryLimit;
+
+    /**
+     * The name
+     *
+     * @param name the name
+     * @return name
+     */
+    @Getter(onMethod = @__(@JsonProperty("name")))
+    private final String name;
+
+    /**
+     * The non basic services allowed
+     *
+     * @param nonBasicServicesAllowed the non basic services allowed boolean
+     * @return the nonBasicServicesAllowed
+     */
+    @Getter(onMethod = @__(@JsonProperty("non_basic_services_allowed")))
+    private final Boolean nonBasicServicesAllowed;
+
+    /**
+     * The quota definition id
+     *
+     * @param organizationQuotaDefinitionId the quota definition id
+     * @return the quota definition id
+     */
+    @Getter(onMethod = @__(@JsonIgnore))
+    private final String organizationQuotaDefinitionId;
+
+    /**
+     * The total private domains
+     *
+     * @param totalPrivateDomains the total private domains
+     * @return the total private domains
+     */
+    @Getter(onMethod = @__(@JsonProperty("total_private_domains")))
+    private final Integer totalPrivateDomains;
+
+    /**
+     * The total routes
+     *
+     * @param totalRoutes the total routes
+     * @return the total routes
+     */
+    @Getter(onMethod = @__(@JsonProperty("total_routes")))
+    private final Integer totalRoutes;
+
+    /**
+     * The total service keys
+     *
+     * @param totalServiceKeys the total service keys
+     * @return the total service keys
+     */
+    @Getter(onMethod = @__(@JsonProperty("total_service_keys")))
+    private final Integer totalServiceKeys;
+
+    /**
+     * The total services
+     *
+     * @param totalServices the total services
+     * @return the total services
+     */
+    @Getter(onMethod = @__(@JsonProperty("total_services")))
+    private final Integer totalServices;
+
+    /**
+     * The trial db allowed
+     *
+     * @param trialDbAllowed the trial db allowed
+     * @return the trial db allowed
+     */
+    @Getter(onMethod = @__(@JsonProperty("trial_db_allowed")))
+    private final Boolean trialDbAllowed;
```

Comment:
@antechrestos Needs to be called `trialDatabaseAllowed` and be `@Deprecated`.  See https://github.com/cloudfoundry/cf-java-client/commit/19d72deeeea040e0aabc1a67581d454801354fd9#commitcomment-17181614